timemachine/fe/single_topology_v3.py

```diff
@@ -508,6 +513,11 @@ def __init__(self, mol_a, mol_b, core, forcefield):
         self.ff = forcefield
         assert core.shape[1] == 2
 
+        a_charge = Chem.GetFormalCharge(mol_a)
```

Comment:
lgtm - I think this is fine. We probably want to actually assert that the nonbonded charges themselves have the same sum later on, but it'd be a pretty egregious bug if nonbonded charges did not sum to the formal charge.

Reply:
We do make the assumption that the `sum(atom_charges) == sum(parameterized_atom_charges)` if that is what you mean?

Reply:
ah great - i forgot